src/components/ChatRoomHeader.tsx

```diff
@@ -0,0 +1,97 @@
+import styled from 'styled-components';
+import { FiChevronLeft } from 'react-icons/fi';
+import { IChatRoomHeaderProps } from './Interface';
+
+const ChatRoomHeader = ({
+  currentUserId,
+  setCurrentUserId,
+}: IChatRoomHeaderProps) => {
+  const handleToggleUser = () => {
+    currentUserId === 'user1'
+      ? setCurrentUserId('user0')
+      : setCurrentUserId('user1');
+  };
+
+  return (
+    <Header>
+      <Button>
+        <FiChevronLeft />
+      </Button>
+
+      <ProfileWrapper onClick={() => handleToggleUser()}>
+        <ProfileImg
+          src={`${process.env.PUBLIC_URL}/imgs/${currentUserId}.jpg`}
+        />
+        <TextWrapper>
+          <UserName>{currentUserId}</UserName>
+          <Typing>Typing...</Typing>
+        </TextWrapper>
+      </ProfileWrapper>
+
+      <Button />
+    </Header>
+  );
+};
+
+const Header = styled.header`
+  padding: 5%;
+  display: flex;
+  justifycontent: space-evenly;
```

Comment:
```suggestion
  justify-content: space-evenly;
```
오타로 인해 해당 코드가 작동하지 않고 있습니다.